script/depots/fluid_depot.lua

```diff
@@ -132,7 +132,7 @@ function fluid_depot:update_contents()
         content.count = box.amount
       end
     end
-    local signal
+    local signal = section.get_slot(1) 
```

Comment:
I think this will some what fix an error for being unble to load the save file.

https://github.com/meglinge/Transport_Drones/blob/e5dd88a60911c72b5ae75de06bbaabd295b7755b/script/depots/buffer_depot.lua#L244-L246